tests/functional_pooling.py

```diff
@@ -27,6 +27,18 @@ def shaparange(s):
 
 
 class TestPooling(unittest.TestCase):
+    def test_average_pooling2d(self):
+        """Test batch_to_space2d and space_to_batch2d."""
+        x = shaparange((2, 3, 10, 30))
+        y = objax.functional.average_pool_2d(x, size=5)
+        z = x.reshape((2, 3, 2, 5, 6, 5)).mean((-3, -1))
+        self.assertEqual(y.tolist(), z.tolist())
+        y = objax.functional.average_pool_2d(x, size=5, strides=1)
```

Comment:
Can we have a test with a small tensor to check the output value of average_pool_2d analytically when the size and strides are different?

Reply:
Done

Reply:
Done